spotbugs/build.gradle

```diff
@@ -65,7 +65,8 @@ dependencies {
   compile 'org.dom4j:dom4j:2.1.1'
   compile 'jaxen:jaxen:1.1.6' // only transitive through dom4j:dom4j:1.6.1, which has an *optional* dependency on jaxen:jaxen.
   compile 'commons-lang:commons-lang:2.6'
-  compile 'org.slf4j:slf4j-api:1.8.0-beta2'
+  compile 'org.slf4j:slf4j-api:1.8.0-beta4'
+  compile 'net.sf.saxon:Saxon-HE:9.9.1-2'
```

Comment:
You should also specify the transformer factory impl to use when requesting it, or it will depend on class path ordering which is used if there is a second XSLT engine included somehow.